texastribune/donations
# Match nav to main repo

#### What's this PR do?

Replaces the old nav with the standardized, queso nav component

#### Why are we doing this? How does it help us?

More seamless experience for a user bouncing between apps

#### How should this be manually tested?

`make backing`
`make`
`make restart` (I think that builds the styles 🤔, if not you'll need to run `npm run css:prod`)

View the nav on each of these pages just for a quick visual check:
- http://local.texastribune.org/ (confirm that donate in the nav shows the active state)
- http://local.texastribune.org/cirlce
- http://local.texastribune.org/business
- http://local.texastribune.org/blastform

#### How should this change be communicated to end users?

I'll let Millie and Jon know about this UX change

#### Are there any smells or added technical debt to note?

Yes, I added a bunch of queso styles to accomplish the nav styles

#### What are the relevant tickets?

https://3.basecamp.com/3098728/buckets/736178/todos/2704839135

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***

* [x] Tested manually on mobile? *(I didn't carry over the mobile menu because I figured there was a chance having Account isolated there might actually be preferred in this app)*

#### TODOs / next steps:
